<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MY LOVELY PAINT V.1</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 206, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="im">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>100</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="file.qrc">:/newPrefix/imIntro.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="newPage">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Console&quot;;
                                    background-color: rgb(248, 217, 255);
                                </string>
       </property>
       <property name="text">
        <string>Создать чистый лист</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openPage">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Console&quot;;
                                    background-color: rgb(248, 217, 255);
                                </string>
       </property>
       <property name="text">
        <string>Открыть файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Console&quot;;
                                    background-color: rgb(248, 217, 255);
                                </string>
       </property>
       <property name="text">
        <string>Информация</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 25, 136);
                        font: 26pt &quot;Fredericka the Great&quot;;
                    </string>
    </property>
    <property name="text">
     <string>MY LOVELY PAINT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
